### DISCUSSION TITLE
Incorporating new machine_config.xml files for Perlmutter

### DISCUSSION THREAD

**Discussion by jacob-stu-allen** [E3SM-Project/E3SM#5513] [E3SM model help]
This is likely a basic and silly question - but the config_machines.xml file that comes from cloning the E3SM source code from git (maint-1.0) does not contain perlmutter as a supported machine. I've tried updating this file and the syslog file with the files [here](https://github.com/E3SM-Project/E3SM/tree/master/cime_config/machines), but the ./create_newcase option fails witha generic xml error:

ImportError: No module named xml.etree.ElementTree

Should the E3SM code from git already have perlmutter as a supported machine?

**Comment by ndkeen** [comment]
I have not yet completed the changes to allow use of the very old maint-1.0 branch on pm-cpu.
https://github.com/E3SM-Project/E3SM/issues/5280

Would also need to make changes for version 1.1 and 1.2
There is no version 1.3
Version 2.0 and 2.1 should be working

**Reply by jacob-stu-allen** [reply]
I've tried cloning maint-1.2 as well, and the issue is the same. 

Cloning maint-1.3 also fails with this error:
fatal: Remote branch maint-1.3 not found in upstream origin

Are there other versions?

**Reply by jacob-stu-allen** [reply]
Thank you - I may have been confused because the "getting started" element of the E3SM website only includes 1.0-1.4 as options. 

I appreciate your help.
